opm/parser/eclipse/Parser/Parser.cpp

```diff
@@ -343,7 +343,7 @@ void ParserState::handleRandomText(const string_view& keywordString ) const {
             << this->current_path()
             << ":" << this->line();
     }
-
+    deck->getMessageContainer().warning(msg.str());
```

Comment:
Here you have access to filename and line number.  Why not add that to the message?